app/lib/frontend/email_sender.dart

```diff
@@ -49,12 +50,18 @@ class EmailSender {
             _server,
             timeout: Duration(seconds: 15),
           ),
-          retryIf: (e) => e is TimeoutException || e is IOException,
+          retryIf: (e) =>
+              e is TimeoutException ||
+              e is IOException ||
+              e is SmtpClientCommunicationException,
           delayFactor: Duration(seconds: 2),
           maxAttempts: 2,
         );
-      } catch (e, st) {
+      } on SmtpMessageValidationException catch (e) {
+        throw EmailSenderException.invalid(e.message);
```

Comment:
Let's not expose raw messages from the SMTP server. I'm not convinced these messages can't contain something sensitive.

I suggest we give a generic reply like "Failed to send email, check that the email address is valid".

Also I think we want to log this.. it could be because our email template contains something that wasn't properly escaped, etc..